modules/ml/src/main/java/org/apache/ignite/ml/genetic/GAGrid.java

```diff
@@ -79,7 +78,6 @@ public GAGrid(GAConfiguration config, Ignite ignite) {
      *
      * @return Average fitness score
      */
-
     private Double calculateAverageFitness() {
 
         double avgFitnessScore = 0;
```

Comment:
Line 105: Do we need return value here?
And the same for line 122, 158, 310, 428

And for 148(+ 404): unnecessary local variable 

Brace style for lines 362 and 365

Reply:
Line 105. Fixed
Line 122, 158, 310, 428. Fixed

148(+ 404): - These variable are necessary.
Brace style for lines 362 and 365. Fixed